contents/blog/posthog-vs-amplitude.md

```diff
@@ -1,5 +1,5 @@
 ---
-date: 2022-05-13
+date: 2023-05-25
```

Comment:
Did you mean to change the date on this one?

Reply:
Nope!